go.mod

```diff
@@ -1,15 +1,20 @@
 module github.com/lightninglabs/neutrino
 
 require (
-	github.com/btcsuite/btcd v0.21.0-beta.0.20201208033208-6bd4c64a54fa
+	github.com/btcsuite/btcd v0.22.0-beta.0.20220111222628-c5e6d3cf5f56
+	github.com/btcsuite/btcd/btcutil v1.0.0
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcutil v1.0.2
+	github.com/btcsuite/btcutil v1.0.2 // indirect
 	github.com/btcsuite/btcwallet/wallet/txauthor v1.0.0
 	github.com/btcsuite/btcwallet/walletdb v1.3.5
 	github.com/btcsuite/btcwallet/wtxmgr v1.3.0
```

Comment:
Just noting that since neutrino **does not require the main btcwallet module**, only `wallet/txauthor`, `wxtmgr`, and `walletdb`, getting rid of the indirect `btcsuite/btcutil` require would require updating those go.mod files too in https://github.com/btcsuite/btcwallet/pull/788#discussion_r791935569